.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### What is the issue
+...
+
+### What does this PR fix and why was it fixed
+...
+
+### Checklist before you submit for review
+- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version
+- [ ] Use `NoSpamLogger` for log lines that may appear frequently in the logs
+- [ ] Verify test results on Butler
+- [ ] Test coverage for new/modified code is > 80%
+- [ ] Proper code formatting
+- [ ] Proper title for each commit staring with the project-issue number, like CNDB-1234
+- [ ] Each commit has a meaningful description
+- [ ] Each commit is not very long and contains related changes
```

Comment:
This doesn't always apply and I don't want people to feel forced to split stuff.

Reply:
let's leave at least "the related changes" part. I hope we can agree on that we don't want a single commit to have unrelated changes